src/ecmult_impl.h

```diff
@@ -405,6 +405,7 @@ static int secp256k1_ecmult_wnaf(int *wnaf, int len, const secp256k1_scalar *a,
     VERIFY_CHECK(0 <= len && len <= 256);
     VERIFY_CHECK(a != NULL);
     VERIFY_CHECK(2 <= w && w <= 31);
+    s = *a;
```

Comment:
nit: this seems like a pretty random location for this line. How about putting it right above the first usage of `s`?

Reply:
I tried to make the change make sense by putting it right after the checks but you're right, in the future this should make sense without knowing there was some change. i'll fix.